src/repos/ClubRepo.ts

```diff
@@ -0,0 +1,37 @@
+import { getConnectionManager, Repository } from 'typeorm';
+import Club from '../entities/Club';
+
+const db = (): Repository<Club> =>
+  getConnectionManager()
+    .get()
+    .getRepository(Club);
+
+const createClub = async (name: string): Promise<void> => {
+  const possibleClub = await db().findOne({ name });
```

Comment:
what is a possibleClub? could this not be just `club`

Reply:
im guessing you're saying the club could not exist.. but if i named all my variables like that i think i'd have `possible` in front of everything

Reply:
i think he wants to create a club iff there isn't one with that name already